tests/visual/listbox.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>ListBox</title>
+
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+</head>
+
+<body>
+
+    <div id="test-area">
+
+        <div class="k-widget k-listbox k-listbox-toolbar-right" deselectable="on" title="" style="">
+            <div class="k-listbox-toolbar">
+                <ul class="k-reset">
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="moveUp" title="Move Up" aria-label="Move Up"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-up"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="moveDown" title="Move Down" aria-label="Move Down"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-down"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferTo" title="Transfer To" aria-label="Transfer To"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-right"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferFrom" title="Transfer From" aria-label="Transfer From"
+                            role="button">
+                            <span class="k-icon k-i-arrow-60-left"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferAllTo" title="Transfer All To" aria-label="Transfer All To"
+                            role="button">
+                            <span class="k-icon k-i-arrow-double-60-right"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="transferAllFrom" title="Transfer All From" aria-label="Transfer All From"
+                            role="button">
+                            <span class="k-icon k-i-arrow-double-60-left"></span>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" class="k-button k-button-icon" data-command="remove" title="Delete" aria-label="Delete" role="button">
+                            <span class="k-icon k-i-x"></span>
+                        </a>
+                    </li>
```

Comment:
While the buttons do inherit styles, showing hovered, focused, pressed and disabled state will be a nice addition.